src/app/static/src/app/components/surveyAndProgram/SurveyAndProgram.vue

```diff
@@ -107,6 +121,9 @@
             };
         },
         computed: {
+            filterTableIndicators(){
+                return this.sapIndicatorsMetadata.filter((val: any) => val.indicator === this.plottingSelections.indicatorId)
```

Comment:
This is the line that's making the tests fail, but I think it's because the Computed sapIndicatorsMetadata prop above in the Computed interface is mistyped. It runs ok, but the test fails to compile that new line. Should be `sapIndicatorsMetadata: ChoroplethIndicatorMetadata` I think?